skimage/transform/_geometric.py

```diff
@@ -570,8 +570,32 @@ class ProjectiveTransform(GeometricTransform):
     params : (D+1, D+1) array
         Homogeneous transformation matrix.
 
-    """
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from skimage.transform import ProjectiveTransform
+    >>> from skimage import data
```

Comment:
```suggestion
    >>> from skimage import data
    >>> from skimage.transform import ProjectiveTransform
```
This is just a detail.